### PR TITLE
Added Cupertino Context Menu in cupertino demo section .

### PR DESCRIPTION
**Pull request by Alabhya268**
Is it ok or anything needs to be changed ?

### CODE REVIEW

**Comment by Alabhya268** [comment]
@rami-a @clocksmith Please have a look at this PR .

**Comment by rami-a** [comment]
Could you please include a screenshot of the demo in action, thanks!

**Comment by Alabhya268** [comment]
> Could you please include a screenshot of the demo in action, thanks!




Please have a look at these screenshots .

**Comment by Alabhya268** [comment]
@rami-a , I just noticed a bug . When app is in dark theme mode ,The text below the flutter icon is not showing up in context menu demo but working as it should when switching to light theme .
bug-2

File: lib/demos/cupertino/cupertino_context_menu_demo.dart [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+class CupertinoContextMenuDemo extends StatelessWidget {
+  const CupertinoContextMenuDemo();
+
+  @override
+  Widget build(BuildContext context) {
+    final galleryLocalizations = GalleryLocalizations.of(context);
+    return CupertinoPageScaffold(
+      navigationBar: CupertinoNavigationBar(
+        middle: Text(galleryLocalizations.demoCupertinoContextMenuTitle),
```
Comment (rami-a) [inline lib/demos/cupertino/cupertino_context_menu_demo.dart:19]: Add `automaticallyImplyLeading: false` so the back button doesn't appear

File: lib/l10n/intl_en.arb [hunk 2]
```diff
@@ -1177,6 +1177,30 @@
...
   },
+  "demoCupertinoContextMenuTitle": "Context Menu",
+  "@demoCupertinoContextMenuTitle": {
+    "description": "Title for the cupertino context menu component demo."
+  },
+  "demoCupertinoContextMenuSubtitle": "iOS-style context menu",
+  "@demoCupertinoContextMenuSubtitle": {
+    "description": "Subtitle for the cupertino context menu component demo."
+  },
+  "demoCupertinoContextMenuDescription": "A full-screen modal route that opens when the child is long-pressed. When open, the CupertinoContextMenu shows the child, or the widget returned by previewBuilder if given, in a large full-screen Overlay with a list of buttons specified by actions.",
```
Comment (rami-a) [inline lib/l10n/intl_en.arb:1188]: Simplify this description a bit . Something like:

"An iOS-style full screen contextual menu that appears when an element is long-pressed."

File: lib/demos/cupertino/cupertino_context_menu_demo.dart (hunk 1 above)
Comment (Alabhya268) [reply lib/demos/cupertino/cupertino_context_menu_demo.dart:19]: ok

**Comment by rami-a** [comment]
> @rami-a , I just noticed a bug . When app is in dark theme mode ,The text below the flutter icon is not showing up in context menu demo ,it should show the text as shown in above screenshots .
> bug-2

Hmm, this is odd, but should be fixable

File: lib/demos/cupertino/cupertino_context_menu_demo.dart (hunk 1 above)
Comment (rami-a) [inline lib/demos/cupertino/cupertino_context_menu_demo.dart:19]: Add a trailing comma to format nicer

File: lib/l10n/intl_en.arb (hunk 2 above)
Comment (Alabhya268) [reply lib/l10n/intl_en.arb]: I've replaced it with this one .

**Comment by Alabhya268** [comment]
@rami-a 

> > @rami-a , I just noticed a bug . When app is in dark theme mode ,The text below the flutter icon is not showing up in context menu demo ,it should show the text as shown in above screenshots .
> > bug-2
> 
> Hmm, this is odd, but should be fixable

should I manually hard-code the text color here ?

**Comment by rami-a** [comment]
Yes, feel free to manually hard code the text color to black here. Additionally, could you center align the text. Then it should be good to merge, thanks!

File: lib/demos/cupertino/cupertino_context_menu_demo.dart [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        children: [
+          Center(
+            child: Container(
+              width: 100,
+              height: 100,
+              child: CupertinoContextMenu(
+                child: Container(
+                  child: const FlutterLogo(size: 250),
+                ),
+                actions: <Widget>[
```
Comment (rami-a) [inline lib/demos/cupertino/cupertino_context_menu_demo.dart:35]: Remove the `<Widget>`

File: lib/demos/cupertino/cupertino_context_menu_demo.dart [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Center(
+            child: Container(
+              width: 100,
+              height: 100,
+              child: CupertinoContextMenu(
+                child: Container(
+                  child: const FlutterLogo(size: 250),
```
Comment (rami-a) [inline lib/demos/cupertino/cupertino_context_menu_demo.dart:33]: The parent container is 100x100 and this is 250, is that going to cause a problem?

File: lib/l10n/intl_en.arb [hunk 5]
```diff
@@ -1177,6 +1177,30 @@
...
+  },
+  "demoCupertinoContextMenuActionOne": "Action one",
+  "@demoCupertinoContextMenuActionOne": {
+    "description": "Context menu list item one"
+  },
+  "demoCupertinoContextMenuActionTwo": "Action two",
+  "@demoCupertinoContextMenuActionTwo": {
+    "description": "Context menu list item two"
+  },
+  "demoCupertinoContextMenuActionText": "Tap and hold flutter logo to see context menu.",
```
Comment (rami-a) [inline lib/l10n/intl_en.arb:1200]:
```suggestion
  "demoCupertinoContextMenuActionText": "Tap and hold the Flutter logo to see the context menu.",
```

**Review by rami-a — APPROVED** [review]
Thanks again for another great demo, looks good but just a few comments

File: lib/demos/cupertino/cupertino_context_menu_demo.dart (hunk 4 above)
Comment (Alabhya268) [reply lib/demos/cupertino/cupertino_context_menu_demo.dart:33]: No it's not causing any problem , The size of flutter icon will be increased to 250 only when context menu will show up .

**Review by rami-a — APPROVED** [review]

**Pull request merged**